src/components/callout/callout.stories.tsx

```diff
@@ -19,25 +12,22 @@ export default {
   },
   argTypes: {
     illustration: {
-      options: Object.keys(EXAMPLE_ILLUSTRATIONS),
-      mapping: EXAMPLE_ILLUSTRATIONS,
+      options: ILLUSTRATION_NAMES,
       description:
-        'This is an example list of icons, there is no default set. You have to provide a svg react component.',
-      defaultValue: 'ShootingStar',
+        'Pick one of the available illustrations. See the Illustration component for more details.',
       control: {
-        type: 'radio',
-        labels: {
-          None: 'None',
-          ShootingStar: 'Shooting Star',
-          ConstellationBigDipper: 'Constellation (Big Dipper)',
-        },
```

Comment:
I removed the manual mapping and I simpky provide the available illustrations which is then passed to the according parameter. 
![Screen Shot 2021-11-16 at 14 57 15](https://user-images.githubusercontent.com/1701755/141998607-d679be84-cb8f-4c63-ba62-60914c2f0393.png)